### PR TITLE
Pm 2022 06 captcheck 3

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by phillimorland** [comment]
@fuzzylogic2000 wel that turned out a lot simpler, hopefully will work on dev

**Comment by phillimorland** [comment]
@fuzzylogic2000 yeah right? if it's working i'll try do a test to up the coverage

File: apps/captcha/assets/captcheck.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ window.onload = function () {
           const questionDiv = document.createElement('div')
           questionDiv.setAttribute('class', 'captcheck_label_message')
           questionDiv.setAttribute('id', 'captcheck_' + idp + '_label_message')
-          questionDiv.innerHTML = "<span class='captcheck_question_image' id='captcheck_" + idp + "_question_image'>" + data.question_i + "</span><span class='captcheck_question_access' id='captcheck_" + idp + "_question_access'>" + data.question_a + "</span><a href='' class='captcheck_alt_question_button' data-prefix='" + idp + "' id='captcheck_" + idp + "_alt_question_button' aria-role='button' aria-label=" + textLabel + " tabindex='0'>" + textTextMode + '</a>'
+          questionDiv.innerHTML = "<span class='captcheck_question_image' id='captcheck_" + idp + "_question_image'>" + data.question_i + "</span><span class='captcheck_question_access' id='captcheck_" + idp + "_question_access'>" + data.question_a + "</span><a href='' class='captcheck_alt_question_button' data-prefix='" + idp + "' id='captcheck_" + idp + "_alt_question_button' aria-role='button' aria-label='" + textLabel + "' tabindex='0'>" + textTextMode + '</a>'
```
Comment (fuzzylogic2000) [inline apps/captcha/assets/captcheck.js:125]: oh wow. good find!

**Review by fuzzylogic2000 — APPROVED** [review]
Supernice!

**Pull request merged**